File: Java/refactoring_exercise_3/src/main/java/za/co/entelect/refactoring3/service/BankingService.java
package za.co.entelect.refactoring3.service;

import za.co.entelect.refactoring3.domain.BankAccount;

import java.util.ArrayList;
import java.util.List;


public class BankingService {

    private List<BankAccount> bankAccounts = new ArrayList<BankAccount>();

    public int countBanksAccounts() {
        return bankAccounts.size();
    }

    public void addBankAccount(BankAccount bankAccount){
        bankAccounts.add(bankAccount);
    }

}

File: C#/Refactoring/refactoring_exercise_3/za/co/entelect/refactoring3/service/BankingService.cs
using System.Collections.Generic;
using refactoring_exercise_3.za.co.entelect.refactoring3.domain;
using refactoring_exercise_3.za.co.entelect.refactoring3.exception;

namespace refactoring_exercise_3.za.co.entelect.refactoring3.service
{

    public class BankingService
    {
        private readonly List<BankAccount> _bankAccounts = new List<BankAccount>();

        public int CountBanksAccounts()
        {
            return _bankAccounts.Count;
        }

        public void AddBankAccount(BankAccount bankAccount)
        {
            _bankAccounts.Add(bankAccount);
        }
    }
    
}